pkg/activator/net/revision_backends_test.go

```diff
@@ -603,8 +604,7 @@ func TestRevisionBackendManagerAddEndpoint(t *testing.T) {
 
 			controller.StartInformers(stopCh, endpointsInformer.Informer())
 
-			rbm := NewRevisionBackendsManagerWithProbeFrequency(stopCh, rt, revisions.Lister(),
-				serviceInformer.Lister(), endpointsInformer, logger, 50*time.Millisecond)
+			rbm := NewRevisionBackendsManagerWithProbeFrequency(ctx, rt, logger, 50*time.Millisecond)
```

Comment:
Why is `stopCh` separate from `ctx.Done()`?

Reply:
This isn't part of your change, but you fixed it below ;-)

Reply:
Done.